main.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+import requests
+import re
+
+
+owner, repo_name = os.environ["OWNER_AND_REPO"].split("/")
```

Comment:
If you're going to make more static content, I suggest creating [a python data class](https://docs.python.org/3/library/dataclasses.html#module-dataclasses).